README.md

```diff
@@ -1,15 +1,22 @@
-Android Key Attestation Sample
+Android Key Attestation Library
 ===================================
 
-This sample illustrates how to use the [Bouncy Castle ASN.1][1] parser to extract information
+This library uses the [Bouncy Castle ASN.1][1] parser to extract information
 from an Android attestation data structure to verify that a key pair has been
-generated in an Android device. 
+generated in an Android device. It is maintained in tandem with Android's key
```

Comment:
Should we say "in Android Keystore" or something more specific?

Maybe "in a hardware-protected environment"?

Reply:
Done.